src/main/java/com/marklogic/contentpump/DocumentMetadata.java

```diff
@@ -70,6 +70,8 @@ public void setNakedProps(boolean isNakedProps) {
      * @param reader
      */
     public static DocumentMetadata fromXML(Reader reader) {
+        xstream.allowTypes(new Class[] {com.marklogic.contentpump.DocumentMetadata.class});
```

Comment:
The parameter that is passed to allowTypes() is an array of classes. So I think you can combine those two lines.